src/contracts/pods/EigenPod.sol

```diff
@@ -50,53 +51,61 @@ contract EigenPod is IEigenPod, Initializable, ReentrancyGuardUpgradeable, Eigen
     /// @notice The single EigenPodManager for EigenLayer
     IEigenPodManager public immutable eigenPodManager;
 
-    /// @notice The amount of eth, in gwei, that is restaked per validator
-    uint64 public immutable REQUIRED_BALANCE_GWEI;
+    ///@notice The maximum amount of ETH, in gwei, a validator can have staked in the beacon chain
+    uint64 public immutable MAX_VALIDATOR_BALANCE_GWEI;
 
-    /// @notice The amount of eth, in wei, that is restaked per ETH validator into EigenLayer
-    uint256 public immutable REQUIRED_BALANCE_WEI;
+    /** 
+    * @notice The value used in our effective restaked balance calculation, to set the 
+    * amount by which to underestimate the validator's effective balance.
+    */
+    uint64 public immutable RESTAKED_BALANCE_OFFSET_GWEI;
 
     /// @notice The owner of this EigenPod
     address public podOwner;
 
     /**
      * @notice The latest block number at which the pod owner withdrew the balance of the pod.
      * @dev This variable is only updated when the `withdraw` function is called, which can only occur before `hasRestaked` is set to true for this pod.
-     * Proofs for this pod are only valid against Beacon Chain state roots corresponding to blocks after the stored `mostRecentWithdrawalBlockNumber`.
+     * Proofs for this pod are only valid against Beacon Chain state roots corresponding to blocks after the stored `mostRecentWithdrawalTimestamp`.
      */
-    uint64 public mostRecentWithdrawalBlockNumber;
+    uint64 public mostRecentWithdrawalTimestamp;
 
     // STORAGE VARIABLES
     /// @notice the amount of execution layer ETH in this contract that is staked in EigenLayer (i.e. withdrawn from the Beacon Chain but not from EigenLayer), 
-    uint64 public restakedExecutionLayerGwei;
+    uint64 public withdrawableRestakedExecutionLayerGwei;
 
     /// @notice an indicator of whether or not the podOwner has ever "fully restaked" by successfully calling `verifyCorrectWithdrawalCredentials`.
     bool public hasRestaked;
 
-    /// @notice this is a mapping of validator indices to a Validator struct containing pertinent info about the validator
-    mapping(uint40 => VALIDATOR_STATUS) public validatorStatus;
```

Comment:
I think you also want to add a deprecated storage slot here as well

Reply:
mm good point. @Sidu28 can we get a comparison on the storage layout / a deprecated slot added here if needed?

Reply:
we're basically replacing unused storage mapping with another one.  


mapping(uint40 => VALIDATOR_STATUS) public validatorStatus becomes mapping(bytes32 => ValidatorInfo) internal _validatorPubkeyHashToInfo;

mapping(uint40 => mapping(uint64 => bool)) public provenPartialWithdrawal; becomes
mapping(bytes32 => mapping(uint64 => bool)) public provenWithdrawal;